jdbc/spring-cloud-starter-stream-sink-jdbc/src/main/resources/META-INF/spring-configuration-metadata-whitelist.properties

```diff
@@ -1 +1,8 @@
 configuration-properties.classes=org.springframework.cloud.stream.app.jdbc.sink.JdbcSinkProperties
+configuration-properties.names=\
+spring.datasource.url,\
+spring.datasource.driver-class-name,\
+spring.datasource.username,\
+spring.datasource.password,\
+spring.datasource.init-sql,\
+spring.datasource.initialize
```

Comment:
Wow! That's cool!
Will fix S3 similar way.

Would you mind to share how does dataflow info look now?
Thanks